packages/pf3-component-mapper/src/tests/__snapshots__/input-addon.test.js.snap

```diff
@@ -567,12 +570,14 @@ exports[`<Input Addon> should render group button before input addon correctly 1
                                                           block={false}
                                                           bsClass="btn"
                                                           bsStyle="default"
+                                                          component="button"
                                                           disabled={false}
                                                           name="set1"
                                                           onClick={[Function]}
                                                         >
                                                           <button
                                                             className="btn btn-default"
+                                                            component="button"
```

Comment:
the component props seem to have sneak to actual DOM elements. We should fix that.